robot-server/robot_server/protocols/router.py

```diff
@@ -306,6 +305,28 @@ async def get_protocols(
     )
 
 
+@protocols_router.get(
+    path="/protocols/ids",
+    summary="Get uploaded protocol ids",
```

Comment:
If I may be a pain in the ass about docs:

```suggestion
    summary="Get uploaded protocol ids",
    description=(
        "Get the IDs of all protocols stored on the server."
        "\n\n"
        "**Warning:**"
        " This is an experimental endpoint and is only meant for internal use by Opentrons."
        " We might remove it or change its behavior without warning."
    ),
```

This does two things.

First, the warning is because we're apparently publicizing these docs now.

Second, providing *anything* as a `description` prevents FastAPI from defaulting to the Python function docstring. We don't want it to do that because it has that "Args: `protocol_store`" internal junk.